packages/backend/src/videos/videos.service.ts

```diff
@@ -1,80 +1,155 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
 import { Repository, getRepository } from 'typeorm';
 import VideosEntity from './videos.entity';
 import { InjectRepository } from '@nestjs/typeorm';
-import {ShortVersionDTO, FilterVideoDTO} from './videos.dto'
+import { ShortVersionDTO, FilterVideoDTO } from './videos.dto';
 import CategoryEntity from './category.entity';
 import * as fs from 'fs';
 
 const shortVersion = Object.keys(new ShortVersionDTO()) as any;
 
+const defaultVideosFolder = "uploads/video/";

```

Comment:
Niech to będzie konfigurowane w env

Reply:
ok @dkolenda ma to być w tym głównym env ?